src/theory/quantifiers/sygus/cegis_unif.cpp

```diff
@@ -329,6 +362,33 @@ void CegisUnifEnumManager::initialize(
   }
   // initialize the current literal
   incrementNumEnumerators();
+  // create single condition enumerator for each decision tree strategy
+  if (options::sygusUnifCondIndependent())
+  {
+    // allocate a condition enumerator for each candidate
+    for (std::pair<const Node, StrategyPtInfo>& ci : d_ce_info)
+    {
+      Node ceu = nm->mkSkolem("cu", ci.second.d_ce_type);
+      // instantiate template for removing redundant operators
+      if (!ci.second.d_sbt_lemma_tmpl[1].first.isNull())
```

Comment:
It seems that some of this is copy/pasted code, can we make this a method that initializes an enumerator with the proper symmetry breaking set up?